CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog for IncidentIo v0.1.0
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## [Unreleased]
```

Comment:
Perhaps link this title to either the tree of the main branch or a diff against the last tag?
```suggestion
## [Unreleased](https://github.com/sgerrand/ex_incident_io/tree/main)
```